Anature/src/application/Player.java

```diff
@@ -122,4 +122,20 @@ public void setSelectedIndex(int selectedIndex)
 
 		mSelectedIndex = selectedIndex;
 	}
+	
+	public boolean canBattle()
+	{
+		boolean canBattle = true;
+		
+		for(Anature anature : mPartyAnature)
+		{
+			if(anature.getCurrHp() <= 0)
```

Comment:
Possible logic error depending on how this is implemented, leaving comment as a reminder for Jesse to check this out!